src/components/NavigationSection/index.tsx

```diff
@@ -39,25 +37,23 @@ const NavigationSection = ({
 
   return (
     <div className={navigationSectionClasses}>
-      <div
+      <button
+        type="button"
         className={titleClassNames}
         onClick={toggle}
-        onKeyDown={toggle}
-        tabIndex={0}
-        role="menuitem"
+        aria-expanded={isOpen}
       >
         <span>
           {isApiDocs && <OfflineBoltIcon />}
           {title}
         </span>
-        {!isMobile &&
```

Comment:
Yes, since I made the sections collapsible on mobile then I wanted the arrows to indicate that the buttons could be toggled, just like on desktop view.